### PR TITLE
Update NexusMods.com.xml

### PR DESCRIPTION
**Pull request by ExE-Boss**
This updates the Nexus Mods ruleset file with the latest data from the Nexus website.

## See also:
- nexus-mods/web-issues#334

## Ruleset will be obsoleted by:
- nexus-mods/web-issues#301

### CODE REVIEW

**Comment by gloomy-ghost** [comment]
Wildcards should be avoided as much as possible unless there is an official statement from the website announcing full https support. Let's wait for nexus-mods/web-issues#301, we have a mechanism to automatically remove rulesets for preloaded domains when it meets our requirement.

**Comment by koops76** [comment]
@gloomy-ghost If the only problematic domain is `wiki`, it's OK to use a wildcard.

**Comment by gloomy-ghost** [comment]
@koops76 There is no effective way for me to check whether _the only problematic domain is wiki_, so I'd rather wait a moment since it's coming soon anyway.

**Comment by J0WI** [comment]
@ExE-Boss would you like to fix the issues addressed by @gloomy-ghost?

**Comment by ExE-Boss** [comment]
@J0WI Given that the primary public subdomains are www, forum, wiki, filedelivery, staticdelivery, static-* and a few others, and the game subdomains have been deprecated with support for them being dropped long before I registered on the Nexus, I’d say, let’s add the `wiki` subdomain once nexus-mods/web-issues#334 is resolved and then wait for the preloaded domain auto-removal once nexus-mods/web-issues#301 is resolved.

File: src/chrome/content/rules/nexusmods.com.xml [hunk 1]
```diff
@@ -5,52 +5,18 @@
...
-	<target host="worldoftanks.nexusmods.com" />
-	<target host="wot.nexusmods.com" />
-	<target host="x.nexusmods.com" />
-	<target host="xcom.nexusmods.com" />
-	<target host="xrebirth.nexusmods.com" />
+	<target host="*.nexusmods.com" />
+
+	<!-- Test URLs -->
+	<test url="http://nexusmods.com/" />
+	<test url="http://www.nexusmods.com/" />
```
Comment (zoracon) [inline src/chrome/content/rules/nexusmods.com.xml]: Since we have self tests against listed hosts, this is a duplicate test url. Please remove this line

**Pull request closed without merging**